Makefile

```diff
@@ -131,3 +131,6 @@ sign-verify-image:
 	@go install github.com/sigstore/cosign/v2/cmd/cosign@v2.1.1
 	COSIGN_PASSWORD=$(COSIGN_PASSWORD) cosign sign --key env://cosign_key $(FULL_IMAGE_NAME) -y
 	cosign verify --key env://cosign_pub $(FULL_IMAGE_NAME)
+
+json-schemas:
```

Comment:
note to self: I think this can be dropped - this was from the original version before we had 'generate' commands to hang this under